h2o-core/src/main/java/water/api/schemas3/ModelMetricsRegressionCoxPHV3.java

```diff
@@ -0,0 +1,30 @@
+package water.api.schemas3;
+
+import hex.ModelMetricsRegressionCoxPH;
+import water.api.API;
+import water.api.API.Direction;
+
+public class ModelMetricsRegressionCoxPHV3 extends ModelMetricsRegressionV3<ModelMetricsRegressionCoxPH, ModelMetricsRegressionCoxPHV3> {
+  @API(help="concordance",direction = Direction.OUTPUT)
```

Comment:
Better description should be provided? At least for stuff like "tiedY"

Reply:
done